src/schema/slack/functions/_scripts/src/templates/test/template_utils_test.ts

```diff
@@ -75,19 +75,60 @@ Deno.test("renderTypeImports should render all imports provided", () => {
         title: "Message time stamp",
         description: "Message time stamp",
       },
+    ],
+  };
+  const actual = renderTypeImports(dfi);
+  assertStringIncludes(actual, "SchemaTypes");
+  assertStringIncludes(actual, "SlackTypes");
+  assertStringIncludes(actual, "InternalSlackTypes");
+});
+
+Deno.test("renderTypeImports should render imports required for array type", () => {
+  const dfi: FunctionRecord = {
+    callback_id: CALLBACK_ID,
+    title: TITLE,
+    description: DESCRIPTION,
+    type: "builtin",
+    input_parameters: [],
+    output_parameters: [
       {
         type: SchemaTypes.array,
         name: "user_ids",
         title: "User Ids",
         description: "User Ids",
         items: {
-          type: "integer",
+          type: SlackTypes.channel_id,
+        },
+      },
+    ],
+  };
+  const actual = renderTypeImports(dfi);
+  assertStringIncludes(actual, "SchemaTypes");
+  assertStringIncludes(actual, "SlackTypes");
+});
+
+Deno.test("renderTypeImports should render imports required for object type", () => {
+  const dfi: FunctionRecord = {
+    callback_id: CALLBACK_ID,
+    title: TITLE,
+    description: DESCRIPTION,
+    type: "builtin",
+    input_parameters: [],
+    output_parameters: [
+      {
+        type: SchemaTypes.object,
+        name: "user_ids",
+        title: "User Ids",
+        description: "User Ids",
+        properties: {
+          my_param: {
+            type: SlackTypes.channel_id,
+          },
         },
       },
```

Comment:
Can you add a test for a function with a mix of different types in its parameters? I.e. one array, one object, and one primitive parameter?

Reply:
Good point 💯  I've added a test that covers the use case where a `primitive` is nested in an `object` that itself is nested in an `array`

lmk if I understood this properly

Reply:
That's a good one too! But i was thinking more of a function with one primitive parameter, a different object parameter, and a third separate array parameter.